### ISSUE TITLE
Initial Input Error vs Date Minimum

### ISSUE THREAD

**Issue by user3587412** [DerekTBrown/inquirer-datepicker-prompt#9]
Currently, when you specify both an initial value and a minimum value, the minimum value takes precedence because the code always sets the `startDate` to the minimum value if a minimum value exists. For example, the below options result in the start date being 2017-01-01 instead of 2017-01-03. 

```
{
  ...
  inital: new Date('2017-01-03'),
  date: {
    min: "1/1/2017",
    max: "3/1/2017"
  }
}
```

I was wondering if it would be possible to set the start date to the initial value instead of the minimum value when both parameters were supplied.

**Comment by DerekTBrown** [comment]
Should be fixed in the latest version, `0.4.2`.

**Comment by user3587412** [comment]
Thank you!
